### PR TITLE
[Localization] Localized "Consent Preferences" in menu-ui-handler.ts

### PR DESCRIPTION
**Pull request by josericardo-fo**
## What are the changes?

Localized the "Consent Preferences" option in the menu

## Why am I doing these changes?

The text was hardcoded

## What did change?

Added locales

### Before (pt_BR)
image

### After (pt_BR)
image

## How to test the changes?

Just open the options menu

## Checklist
- [X] **I'm using `beta` as my base branch**
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?
- [X] Have I considered writing automated tests for the issue?
- [X] Have I tested the changes (manually)?
    - [X] Are all unit tests still passing? (`npm run test`)
- [X] Are the changes visual?
  - [X] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/fr/menu-ui-handler.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ export const menuUiHandler: SimpleTranslationEntries = {
   "exportSlotSelect": "Sélectionnez l’emplacement depuis lequel exporter les données.",
   "importData": "Importer données",
   "exportData": "Exporter données",
+  "consentPreferences": "Consent Preferences",
```
Comment (Adri1) [inline src/locales/fr/menu-ui-handler.ts:21]:
```suggestion
  "consentPreferences": "Préférences des cookies",
```

File: src/locales/de/menu-ui-handler.ts [hunk 2]
```diff
@@ -18,6 +18,7 @@ export const menuUiHandler: SimpleTranslationEntries = {
   "exportSlotSelect": "Wähle einen Slot zum Exportieren.",
   "importData": "Daten importieren",
   "exportData": "Daten exportieren",
+  "consentPreferences": "Consent Preferences",
```
Comment (CodeTappert) [inline src/locales/de/menu-ui-handler.ts:21]:
```suggestion
  "consentPreferences": "Einwilligungspräferenzen",
```

File: src/locales/it/menu-ui-handler.ts [hunk 3]
```diff
@@ -16,12 +16,13 @@ export const menuUiHandler: SimpleTranslationEntries = {
   "importSlotSelect": "Seleziona uno slot in cui importare.",
   "exportSession": "Esporta sessione",
   "exportSlotSelect": "Seleziona uno slot da cui esportare.",
+  "importData": "Importa dati",
+  "exportData": "Esporta dati",
+  "consentPreferences": "Consent Preferences",
   "linkDiscord": "Link Discord",
   "unlinkDiscord": "Unlink Discord",
   "linkGoogle": "Link Google",
   "unlinkGoogle": "Unlink Google",
```
Comment (NicusPulcis) [inline src/locales/it/menu-ui-handler.ts:25]:
```suggestion
  "importData": "Importa dati",
  "exportData": "Esporta dati",
  "consentPreferences": "Consenti preferenze",
  "linkDiscord": "Collega Discord",
  "unlinkDiscord": "Scollega Discord",
  "linkGoogle": "Collega Google",
  "unlinkGoogle": "Scollega Google",
```

**Review by NicusPulcis — APPROVED** [review]
italian localization

File: src/locales/fr/menu-ui-handler.ts [hunk 4]
```diff
@@ -18,6 +18,7 @@ export const menuUiHandler: SimpleTranslationEntries = {
   "exportSlotSelect": "Sélectionnez l’emplacement depuis lequel exporter les données.",
   "importData": "Importer données",
   "exportData": "Exporter données",
+  "consentPreferences": "Préférences des cookies",
```
Comment (Adri1) [inline src/locales/fr/menu-ui-handler.ts:21]:
```suggestion
  "consentPreferences": "Gérer les cookies",
```

**Review by Adri1 — COMMENTED** [review]
French adjustment

File: src/locales/es/menu-ui-handler.ts [hunk 5]
```diff
@@ -18,6 +18,7 @@ export const menuUiHandler: SimpleTranslationEntries = {
   "exportSlotSelect": "Selecciona una ranura para exportar.",
   "importData": "Importar Datos",
   "exportData": "Exportar Datos",
+  "consentPreferences": "Consent Preferences",
```
Comment (AsdarDevelops) [inline src/locales/es/menu-ui-handler.ts:21]:
```suggestion
  "consentPreferences": "Consentimiento de datos",
```

**Review by AsdarDevelops — COMMENTED** [review]
ES

File: src/locales/zh_CN/menu-ui-handler.ts [hunk 6]
```diff
@@ -18,6 +18,7 @@ export const menuUiHandler: SimpleTranslationEntries = {
   "exportSlotSelect": "选择要导出的存档位。",
   "importData": "导入数据",
   "exportData": "导出数据",
+  "consentPreferences": "Consent Preferences",
```
Comment (RimKnight) [inline src/locales/zh_CN/menu-ui-handler.ts:21]:
```suggestion
  "consentPreferences": "同意偏好",
```

**Review by RimKnight — APPROVED** [review]

File: src/locales/ko/menu-ui-handler.ts [hunk 7]
```diff
@@ -18,6 +18,7 @@ export const menuUiHandler: SimpleTranslationEntries = {
   "exportSlotSelect": "내보낼 슬롯을 골라주세요.",
   "importData": "데이터 불러오기",
   "exportData": "데이터 내보내기",
+  "consentPreferences": "Consent Preferences",
```
Comment (EnochG1) [inline src/locales/ko/menu-ui-handler.ts:21]:
```suggestion
  "consentPreferences": "쿠키 설정 동의",
```

**Review by Adri1 — APPROVED** [review]

**Review by EnochG1 — APPROVED** [review]

**Pull request merged**